app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -0,0 +1,106 @@
+  protected @Bind(R.id.creator_dashboard_project_name) TextView projectNameTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining) TextView timeRemainingTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining_text) TextView timeRemainingTextTextView;
+  protected @BindString(R.string.discovery_baseball_card_stats_pledged_of_goal) String pledgedOfGoalString;
+
+  private KSCurrency ksCurrency;
+  private KSString ksString;
+
+  @Override
+  protected void onCreate(final @Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.creator_dashboard_layout);
+    ButterKnife.bind(this);
+
+    this.ksCurrency = this.environment().ksCurrency();
+    this.ksString = this.environment().ksString();
+
+    viewModel.outputs.latestProject()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setPledgedOfGoalString);
+
+    viewModel.outputs.latestProject()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setTimeRemainingTextTextView);
+
+    viewModel.outputs.timeRemaining()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(timeRemainingTextView::setText);
+
+    viewModel.outputs.projectBackersCountText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(backerCountTextView::setText);
+
+    viewModel.outputs.projectNameTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(projectNameTextView::setText);
+
+    viewModel.outputs.startProjectActivity()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(projectAndRefTag -> this.startProjectActivity(projectAndRefTag.first, projectAndRefTag.second));
+  }
+
+  @OnClick({ R.id.creator_view_project_button, R.id.creator_dashboard_project_context_view })
+  protected void viewProjectButtonClicked() {
+    this.viewModel.inputs.projectViewClicked();
+  }
+
+  private void startProjectActivity(final @NonNull Project project, final @NonNull RefTag refTag) {
+    final Intent intent = new Intent(this, ProjectActivity.class)
+      .putExtra(IntentKey.PROJECT, project)
+      .putExtra(IntentKey.REF_TAG, refTag);
+    startActivity(intent);
+  }
+
+  private void setPledgedOfGoalString(final @NonNull Project latestProject) {
+    final String goalString = ksCurrency.format(latestProject.pledged(), latestProject, false, true, RoundingMode.DOWN);
+    amountRaisedTextView.setText(goalString);
+
+    final String goalText = ksString.format(this.pledgedOfGoalString, "goal", goalString);
+    fundingTextTextView.setText(goalText);
+  }
+
+  private void setTimeRemainingTextTextView(final @NonNull Project latestProject) {
+    timeRemainingTextTextView.setText(ProjectUtils.deadlineCountdownDetail(latestProject, this.getApplicationContext(), ksString));
```

Comment:
since `this` is an Activity, which inherently is a `context`, you can replace `this.getApplicationContext()` with just `this`!